runelite-client/src/main/java/net/runelite/client/plugins/discord/DiscordGameEventType.java

```diff
@@ -262,7 +294,95 @@
 
 	// Raids
 	RAIDS_CHAMBERS_OF_XERIC("Chambers of Xeric", DiscordAreaType.RAIDS, Varbits.IN_RAID),
-	RAIDS_THEATRE_OF_BLOOD("Theatre of Blood", DiscordAreaType.RAIDS, Varbits.THEATRE_OF_BLOOD);
+	RAIDS_THEATRE_OF_BLOOD("Theatre of Blood", DiscordAreaType.RAIDS, Varbits.THEATRE_OF_BLOOD),
+
+	// Other
+	REGION_ABYSSAL_AREA("Abyssal Area", DiscordAreaType.REGIONS, 12108),
+	REGION_ABYSSAL_NEXUS("Abyssal Nexus", DiscordAreaType.REGIONS, 12106),
+	REGION_AGILITY_PYRAMID("Agility Pyramid", DiscordAreaType.REGIONS, 13356),
+	REGION_AIR_ALTAR("Air Altar", DiscordAreaType.REGIONS, 11339),
+	REGION_AL_KHARID_MINE("Al Kharid Mine", DiscordAreaType.REGIONS, 13107),
+	REGION_APE_ATOLL("Ape Atoll" , DiscordAreaType.REGIONS, 10795, 10974, 11050),
+	REGION_ARANDAR("Arandar", DiscordAreaType.REGIONS, 9266, 9267, 9523),
+	REGION_ASGARNIA("Asgarnia", DiscordAreaType.REGIONS, 11825, 11829, 11830, 12085, 12086),
+	REGION_BATTLEFIELD("Battlefield", DiscordAreaType.REGIONS, 10034),
+	REGION_BATTLEFRONT("Battlefront", DiscordAreaType.REGIONS, 5433, 5434),
+	REGION_BLAST_MINE("Blast Mine", DiscordAreaType.REGIONS, 5948),
+	REGION_BODY_ALTAR("Body Altar", DiscordAreaType.REGIONS, 10059),
+	REGION_CHAOS_ALTAR("Chaos Altar", DiscordAreaType.REGIONS, 9035),
+	REGION_COSMIC_ALTAR("Cosmic Altar", DiscordAreaType.REGIONS, 8523),
+	REGION_COSMIC_ENTITYS_PLANE("Cosmic Entity's Plane", DiscordAreaType.REGIONS, 8267),
+	REGION_CRABCLAW_ISLE("Crabclaw Isle", DiscordAreaType.REGIONS, 6965),
+	REGION_CRAFTING_GUILD("Crafting Guild", DiscordAreaType.REGIONS, 11571),
+	REGION_CRANDOR("Crandor", DiscordAreaType.REGIONS, 11314, 11315),
+	REGION_CRASH_ISLAND("Crash Island", DiscordAreaType.REGIONS, 11562),
+	REGION_DARK_ALTAR("Dark Altar", DiscordAreaType.REGIONS, 6716),
+	REGION_DEATH_ALTAR("Death Altar", DiscordAreaType.REGIONS, 8779),
+	REGION_DEATH_PLATEAU("Death Plateau", DiscordAreaType.REGIONS, 11320),
+	REGION_DENSE_ESSENCE("Dense Essence Mine", DiscordAreaType.REGIONS, 6972),
+	REGION_DRAGONTOOTH("Dragontooth Island", DiscordAreaType.REGIONS, 15159),
+	REGION_EAGLES_PEAK("Eagles' Peak", DiscordAreaType.REGIONS, 9270),
+	REGION_EARTH_ALTAR("Earth Altar", DiscordAreaType.REGIONS, 10571),
+	REGION_ENCHANTED_VALLEY("Enchanted Valley", DiscordAreaType.REGIONS, 12102),
+	REGION_FALADOR_FARM("Falador Farm", DiscordAreaType.REGIONS, 12083),
+	REGION_FARMING_GUILD("Farming Guild", DiscordAreaType.REGIONS, 4922),
+	REGION_FELDIP_HILLS("Feldip Hills", DiscordAreaType.REGIONS, 9773, 9774, 10029, 10030, 10285, 10286, 10287, 10542),
```

Comment:
A tiny bit of land sticks into this region
```suggestion
	REGION_FELDIP_HILLS("Feldip Hills", DiscordAreaType.REGIONS, 9773, 9774, 10029, 10030, 10285, 10286, 10287, 10542, 10543),
```